### DISCUSSION TITLE
A question about loading multiple pretrained models.

### DISCUSSION THREAD

**Discussion by EdyQiu** [open-mmlab/mmclassification#1405] [Q&A] [answered]
Hi, I have a question about loading multiple pre-trained models. 
First, I pre-trained three ResNet models respectively, and then I will perform joint training(feature ensemble).
So, I am here asking for any suggestions about how I should modify the code(like which file/function should I modify)

Any suggests are very welcome! Thank you!

**Comment by Ezra-Yu** [comment, marked as answer]
You can use init_cfg in each sub-module, for example:

```
model = dict(
    type='CustomImageClassifier',
    backbone1=dict(
        init_cfg=dict(
            type='Pretrained', checkpoint=pretrained1, prefix='backbone')),
    backbone2=dict(
        init_cfg=dict(
            type='Pretrained', checkpoint=pretrained2, prefix='backbone')),
    backbone3=dict(
        init_cfg=dict(
            type='Pretrained', checkpoint=pretrained3, prefix='backbone')),
    head=dict(num_classes=200, )
)
```

**Reply by EdyQiu** [reply]
Thanks for answering the question! 
This seems to be a much more direct way than combing the .pkl file into a bigger one (which I initially thought haha).
let me try to implement this framework.
